hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysRenameRequest.java

```diff
@@ -276,4 +281,16 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     auditMap.put(UNRENAMED_KEYS_MAP, unRenameKeysMap.toString());
     return auditMap;
   }
+
+  public static OMKeysRenameRequest getInstance(
+      OzoneManagerProtocolProtos.RenameKeysArgs keyArgs, OMRequest omRequest,
+      OzoneManager ozoneManager) throws IOException {
+
+    BucketLayout bucketLayout =
+        OzoneManagerUtils.getBucketLayout(keyArgs.getVolumeName(),
+            keyArgs.getBucketName(), ozoneManager, new HashSet<>());
+    Preconditions.checkArgument(!bucketLayout.isFileSystemOptimized(),
```

Comment:
Same comment as keysDeleteRequest.

Reply:
Thanks noted, I will address these assertions separately in [HDDS-6426](https://issues.apache.org/jira/browse/HDDS-6426) .